.github/ISSUE_TEMPLATE/snmp-traps-profile-request.md

```diff
@@ -0,0 +1,57 @@
+---
+name: SNMP Traps Profile Request
+about: Request a new Traps profile be created for a device
+title: "[Missing SNMP Traps Profile]"
+labels: snmp-trap-profile
+assignees: ''
+
+---
+
+[NOTE]: # ( ^^ Please add the Device Vendor and Model in the title above. ^^ )
+
+Device Vendor: 
+
+Device Model: 
+
+[NOTE]: # (If you do not know your SysOID, it can be gathered from the SNMP walk you provide at the end of this issue. It is the value of this OID in the SYSTEM-MIB: `1.3.6.1.2.1.1.2.0`)
+
+System Object Identifier (SysOID): 
+
+List of critical metrics that you expect in the profile (does not need to be all-inclusive, just a general idea of your expectations):
```

Comment:
Does this make sense for traps? We are just going to parse all incoming varbinds aren't we?